scout/server/blueprints/variant/templates/variant/gene_disease_relations.html

```diff
@@ -19,13 +19,46 @@
                   {% if disease_term.source == 'OMIM' %}
                     <tr>
                       <td>
-                          <a href="http://omim.org/entry/{{ gene.common.omim_id }}" target="_blank">
+                          <a href="http://omim.org/entry/{{ gene.common.omim_id }}" rel="noopener" target="_blank">
                             {{ gene.common.hgnc_symbol }}
                           </a>
                       </td>
                       <td>
 
-                          <a href="http://omim.org/entry/{{ disease_term._id|replace('OMIM:', '') }}" target="_blank">
+                          <a href="{{ disease_term.disease_link }}" rel="noopener" target="_blank">
```

Comment:
Nice!

<img width="557" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/b21d5a58-9eff-4cec-8e95-e865033a0a69">

Reply:
Again, if you look instead on the gene page and you search APC you only get one link to diseases. This is probably a bug we have in current main

<img width="1680" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/0d7a7c3f-64e9-4d3e-9a0a-cea4fd2ea8be">

Reply:
There are 6 diseases linked in your picture, the disease links are in the table to the right with the heading "Phenotypes". The link in the middle table is for the gene in omim.

Reply:
Ahhh yes, Phenotyoes instead of diseases. OK!

Reply:
We could add also the ORPHA ones to the table and rename to diseases perhaps?It's also OK to fix in another PR

Reply:
It was listed as that before but considering the renaming in the other tables, I think it would be best to do the same here perhaps and change it ti "Diseases"?

Reply:
Agred! 👍🏻

Reply:
The table heading has been changed and the genes found